app/src/lib/stores/accounts-store.ts

```diff
@@ -182,7 +183,8 @@ export class AccountsStore extends BaseStore {
         account.emails,
         account.avatarURL,
         account.id,
-        account.name
+        account.name,
+        account.scopes || []
```

Comment:
Actually you do what I just mentioned ☝️, so maybe we can get rid of the optional bit for `scopes`.

Reply:
This bit is important because above here we're parsing raw JSON into `IAcccount[]` - and those might be existing . So for backwards-compatibility I'd like to keep that around.

Reply:
Actually, I'm going to drop this from review and see if there's a chance to migrate this data properly. I want `IAccount` to be as accurate as possible here, and leaving optional fields around does feel like a code smell.